modules/lobbies/scripts/find_or_create.ts

```diff
@@ -34,6 +37,15 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
+	await ctx.modules.captcha.guard({
+		key: "lobbies.find_or_create",
+		period: 15,
```

Comment:
period & requests should be configurable in config.ts and have defaults in the default config